lib/api/v1/fetching-controller.js

```diff
@@ -5,10 +5,10 @@ var app = express();
 var auth = require('../authentication')(app);
 var user = require('../authorization')(app, auth);
 var logger = require('../../logger');
+var prefs = require('../../../config/server-prefs.json');
```

Comment:
Did you check with Aman about splitting this out? I know he had some thoughts on the matter.